<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnListele">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Listele</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ÖĞRENCİ KİTAP SEÇİMİ</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_OgrenimDurumu">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Öğrenim Durumu</string>
   </property>
   <widget class="QRadioButton" name="rdBtn_Lise">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lise</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdBtn_Universite">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Üniversite</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_Lise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Lise</string>
   </property>
   <widget class="QRadioButton" name="rdbtnSnf1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnSnf2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnSnf3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnSnf4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4.Sınıf</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_Uni">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Üniversite</string>
   </property>
   <widget class="QRadioButton" name="rdbtnOnL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ön Lisans</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnLisans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisans</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnYLisans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yüksek Lisans</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_LiseK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Lise Kitapları</string>
   </property>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AYT Kitapları</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TYT Kitapları</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>YKS Denemeler</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_UniK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Üniversite Kitapları</string>
   </property>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kpss Kitapları</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ales Kitapları</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>YDS Kitapları</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>200</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ad:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Soyad:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Numara:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lnAd"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lnSoyad"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lnNumara"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
